common/build/eslint-plugin-fluid/package.json

```diff
@@ -0,0 +1,59 @@
+{
+	"name": "@fluid-internal/eslint-plugin-fluid",
+	"version": "0.1.0",
+	"private": true,
+	"description": "Custom ESLint rules for the Fluid Framework",
+	"homepage": "https://fluidframework.com",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/microsoft/FluidFramework.git",
+		"directory": "common/build/eslint-pluig-fluid/"
+	},
+	"license": "MIT",
+	"author": "Microsoft and contributors",
+	"main": "index.js",
+	"scripts": {
+		"build": "npm run build:readme",
+		"build:readme": "markdown-magic --files \"**/*.md\"",
+		"clean": "rimraf --glob nyc",
+		"cleanup-printed-configs": "node ./scripts/cleanup-printed-configs.js ./printed-configs",
```

Comment:
This one isn't necessary in this package. eslint-config-fluid uses it to print full version of the configs for reference but here we don't have to do that.

Reply:
I thought we still want to keep the `build` script since we want to keep the `build` parameter in `tools/pipelines/build-eslint-plugin-fluid.yml`.

Reply:
Yes, I mean specifically cleanup-printed-configs. Github's UI just gave a bit more context for the comment :)